be/src/vec/functions/function_json.cpp

```diff
@@ -1064,6 +1064,187 @@ class FunctionJsonContains : public IFunction {
     }
 };
 
+class FunctionJsonMergePatch : public IFunction {
+public:
+    static constexpr auto name = "json_merge_patch";
+    static FunctionPtr create() { return std::make_shared<FunctionJsonMergePatch>(); }
+
+    String get_name() const override { return name; }
+
+    size_t get_number_of_arguments() const override { return 0; }
+
+    bool is_variadic() const override { return true; }
+
+    DataTypePtr get_return_type_impl(const DataTypes& arguments) const override {
+        return make_nullable(std::make_shared<DataTypeString>());
```

Comment:
Does it need to be always_nullable? for not_nullable inputs, it seems will not generate null value?